_posts/2020/08/2020-08-31-governance.md

```diff
@@ -0,0 +1,65 @@
+---
+title: "Governance"
+date: 2020-08-31
+---
+
+I had another good conversation with [Mike Hoye](http://exple.tive.org/blarg/) this week
+about life, the Internet, and everything.
+I always learn a lot talking with him,
+and what I realized this time is that *governance should be discoverable*.
+Pretty much every open source project has a `LICENSE.md` file in their root directory these days,
+and most now have a `CONDUCT.md` file as well.
+These files tell passers-by how they're allowed to use the software
+and what social norms they're required to adhere to if they want to contribute to it.
+
+But how can they find out who decides what the project is going to do next,
+or who gets a say in deciding that?
```

Comment:
I'd say that even the current status and the things the project cares about NOW are important. 
It is important to be very transparent, to avoid issues like people contributing things that are not part of the vision of the project, and end up not accepted, leading to demotivation.  We found something like this here: http://igor.pro.br/publica/papers/ICSE_2018b.pdf (look for "mismatch between developer’s and team’s vision")